core/src/banking_stage.rs

```diff
@@ -628,11 +632,18 @@ impl BankingStage {
         // Spawn the central scheduler thread
         bank_thread_hdls.push({
             let packet_deserializer = PacketDeserializer::new(non_vote_receiver);
+            let receive_and_buffer = SanitizedTransactionReceiveAndBuffer::new(
+                packet_deserializer,
+                bank_forks.clone(),
+                forwarder.is_some(),
+            );
+            let container = TransactionStateContainer::with_capacity(TOTAL_BUFFERED_PACKETS);
```

Comment:
Once the other structures/impl for new transaction type are ready there will be a switch here on spawning.
- Receiving and buffering is generic so that we can skip over all deserialization (never pass through `ImmutableDeserializedPacket`)
- Container is generic because the current impl isn't good if we want to copy as little data as possible. For new tx types, it will wrap the old type with a pre-allocated (separate) container for getting free `Bytes` spaces to copy our packet data into. Eventually this would go away once we have `Bytes` upstream, but for now we don't and this is much faster than allocating space or copying a `Packet` for every tx move.